src/UI/UIView.cpp

```diff
@@ -790,6 +802,11 @@ void UIView::draw_ui_callback()
 
       if (helper::ends_with_string(fileDialog.selected_fn, ".txt"))
       {
+        //  TO DO:  Thread the data loading process to run in the background and implement loading UI component
```

Comment:
If you leave the to do as one word (`// TODO: `) VS code has plenty of extensions to change the color of the comment. Not necessary at all but figured I'd share

Reply:
I changed them to TODO. 👍